pulseguardian/management.py

```diff
@@ -9,106 +9,106 @@
 
 import requests
 
+from pulseguardian import config
 MAX_RETRY = 5
 
 
 class PulseManagementException(Exception):
     pass
 
+"""
+Wrapper around the RabbitMQ management plugin's REST API.
+"""
```

Comment:
Please make this a standard function doc string (in this case, all one line, as the first line in the function definition).